src/main/java/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.java

```diff
@@ -1812,9 +1812,8 @@ public DefaultShardManagerBuilder setWebsocketFactory(@Nullable WebSocketFactory
 
     /**
      * The {@link ChunkingFilter} to filter which guilds should use member chunking.
-     * <br>By default this uses {@link ChunkingFilter#ALL}.
      *
-     * <p>If a guild is configured for chunking the {@link #setMemberCachePolicy(MemberCachePolicy)} will be ignored.
+     * <p>Use {@link #setMemberCachePolicy(MemberCachePolicy)} to configure which members to keep in cache from chunking.
```

Comment:
I'm noticing that the `setChunkingFilter` for DefaultShardManagerBuilder is different in behavior to that of JDABuilder. JDABuilder sets ChunkingFilter.ALL. Is there a reason the shard manager doesn't?

Reply:
The behavior is the same since you can't create one without setting a chunking filter